typesafe_config_base/README.md

```diff
@@ -0,0 +1,16 @@
+# wellcome/typesafe_config_base
+
+Base container image for typesafe services at Wellcome.
```

Comment:
What is a “typesafe service”?

How about “for Scala apps that use Typesafe (link) for config”?